music_assistant/providers/spotify/provider.py

```diff
@@ -139,33 +178,45 @@ async def search(
                 "search", q=search_query, type=searchtype, limit=page_limit, offset=offset
             )
             if "artists" in api_result:
-                searchresult.artists += [
+                new_artists = [
```

Comment:
whats up with the new_ prefix ?

Reply:
Can't remember. Must have been a leftover from earlier in the development... Removed.